app/src/main/res/layout/selection_interaction_item.xml

```diff
@@ -19,24 +23,37 @@
     android:descendantFocusability="beforeDescendants"
     android:focusableInTouchMode="true"
     android:orientation="vertical"
-    android:padding="0dp"
+    android:paddingTop="12dp"
+    android:paddingBottom="12dp"
```

Comment:
**Comment 5**

Keeping `Radio` button UI in mind where we won't show the below textview, as well as taking **comment 3** as reference where we divided `24dp` in `12dp` top and `12dp` bottom.
Because the top and the bottom item of RecyclerView will have only `12dp` gap from border we are adding this `12dp` paddingTop and paddingBottom to make sure that overall gap between the top/bottom item from its top/bottom border is `24dp`.